src/metabase/search/scoring.clj

```diff
@@ -81,31 +81,29 @@
                 {:is_match is-match
                  :text     (tokens->string text-tokens (not is-match))})))))
 
-(defn- text-score-with
+(defn- text-scores-with
   "Scores a search result. Returns a map with the score and other info about the text match,
    if there is one. If there is no match, the score is 0."
   [weighted-scorers query-tokens search-result]
-  (let [total-weight (reduce + (map :weight weighted-scorers))
-        scores       (for [column (search-config/searchable-columns-for-model (:model search-result))
-                           :let   [matched-text (-> search-result
-                                                    (get column)
-                                                    (search-config/column->string (:model search-result) column))
-                                   match-tokens (some-> matched-text normalize tokenize)
-                                   score        (and matched-text
-                                                     (reduce (fn [tally f]
-                                                               (+ tally
-                                                                  (f query-tokens match-tokens)))
-                                                             0
-                                                             (map :scorer weighted-scorers)))]
-                           :when  (and matched-text
-                                       (pos? score))]
-                       {:score               (/ score total-weight)
+  ;; TODO is pmap over search-result worth it?
+  (let [scores       (for [column      (search-config/searchable-columns-for-model (:model search-result))
+                           {:keys [scorer name weight]
+                            :as   _ws} weighted-scorers
```

Comment:
Nit, why bother with the `:as _ws` ?

Reply:
That is a trick I use to keep the alignment of the r.h.s. of the for comprehension from being too far over to the right

<img width="1060" alt="Screen Shot 2022-10-27 at 4 24 43 PM" src="https://user-images.githubusercontent.com/343288/198409577-508b4c0d-45b9-4200-b5a4-146db4a668e0.png">

If you think it's too weird I'm open to nixing it.